### PR TITLE
Added path to view individual review cards

### PR DESCRIPTION
**Pull request by DJHymers**

### CODE REVIEW

File: src/components/SingleReview.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const [review, setReview] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetchReviewById(review_id).then((review) => {
+      setReview(review);
+      setIsLoading(false);
+    });
+  }, [review_id]);
```
Comment (lewispricey) [inline src/components/SingleReview.jsx:16]: Awesome to see the reviewId in your dependency array to ensure the useEffect triggers on manual url update!

**Review by lewispricey — CHANGES_REQUESTED** [review]
Great work on this ticket so far David!

Your codes super clean and easy to read which is awesome and its great to see the async/await in your api calls 👍 

Both the reviews and singleReview pages are not currently responsive and are currently exceeding the screen in mobile view.

Please could you tweak your CSS to make these responsive and submit another PR once you're ready (I'll send a screenshot over on Slack)

**Review by alexdevdavis — CHANGES_REQUESTED** [review]
The responsive sizing is better for the single review page, but the reviews list doesn't change at all depending on screen size. 

- Consider working with `width` and/or `max-width` to prevent the site moving beyond the viewport

**Review by lewispricey — APPROVED** [review]
Great stuff David, thanks for implementing the feedback and making the page responsive.

Good to merge when you're ready 🚀

**Pull request merged**